tcms/testruns/migrations/0004_squashed.py

```diff
@@ -480,4 +480,4 @@ class Migration(migrations.Migration):
                 "get_latest_by": "history_date",
             },
         ),
-    ]
+    ]
```

Comment:
Alright, I finally figured out what's going on here. Line 114 is `TestRun.stop_date`. That field is unrelated to the current issue and it has the appropriate name from the start. No changes should be made there.

Line 397 is similar but for HistoricalTestRun. 

@APiligrim  I've made a confusing comment before but you have overlooked the fact that these migration operations are towards a completely different model.

Reply:
Revert all changes to this file, it should not show up in the diff at all.

Reply:
Thank you! I reverted my changes to that specific file back to be like what is on master 👍